OpenSim/Examples/ControllerExample/ControllerExample.cpp

```diff
@@ -317,8 +318,11 @@ int main()
         manager.integrate( si );
 
         // Save the simulation results.
-        osimModel.printControlStorage( "tugOfWar_controls.sto" );
-        manager.getStateStorage().print( "tugOfWar_states.sto" );
+        auto controlsTable = osimModel.getControlsTable();
+        MOTFileAdapter::write(controlsTable, "tugOfWar_controls.mot");
```

Comment:
To summarize, I would like us to:
 1. revert changes to `.mot` back to `.sto`
 2. create a stand in for STOFileAdpater (e.g. aliasing MOTFileAdapter is good enough). 
 
I am happy for you to create the separate STO/MOT adapters as a future PR. :crystal_ball: